### DISCUSSION TITLE
Is there a way to completely turn off battery notifications?

### DISCUSSION THREAD

**Discussion by ghost** [AntiMicroX/antimicrox#1030] [Q&A] [answered]
I'm using DS4 controllers and AntiMicroX has been spamming low battery notification windows every ~15-20 minutes. I'm pretty sure this is a Linux or a Sony battery charge reporting issue but it has annoyed me enough to want to turn these notifications off completely.  I  looked through the app's settings and options and didn't find any toggle for it. Is there a method to turn these notifications off?

**Comment by pktiuk** [comment]
Check the settings https://github.com/AntiMicroX/antimicrox/pull/146

**Reply by ghost** [reply]
That setting tab ends at the 'Attach the numeric keypad' setting on my end. I'm not sure what is the reason behind it. I tried to check it while the controller was connected and disconnected and did not see both of the notification settings in https://github.com/AntiMicroX/antimicrox/pull/146.

![Screenshot from 2024-08-30 20-48-38](https://github.com/user-attachments/assets/e911f4c4-fa4c-42ad-976e-98de24ee3da5)

**Reply by ghost** [reply, marked as answer]
A reinstall fixed the issue and I can now see and interact with the notification settings!

Thanks for the help and for making such an amazing app. It has been very helpful since I switched to Linux a few months back!
